### PR TITLE
make show, edit and new transaction functions, fixed payee_id in import ...

### PR DESCRIPTION
**Pull request by sramallo**
...and fixed account_id nil

### CODE REVIEW

File: app/views/transactions/_form.html.erb [hunk 1]
```diff
@@ -1,65 +1,65 @@
 <%= form_for(@transaction) do |f| %>
```
Comment (framallo) [inline app/views/transactions/_form.html.erb:1]: you should use haml

Comment (sramallo) [inline app/views/transactions/_form.html.erb:1]: Yes, I know, but first I want to make it work and then pass it to haml.

Wath do you think about?

```
= link_to t('.show', :default => t("helpers.links.show")), transaction,
```

:class => 'btn btn-mini btn-info'
    %i.icon-white

I have a question with this.
Why you think that I need to put the _do_ in:

= link_to t('.show', :default => t("helpers.links.show")), transaction,
:class => 'btn btn-mini btn-info'  do
    %i.icon-white

in this way does not work.

Thanks
SR

2013/12/10 Federico Ramallo notifications@github.com

File: app/views/transactions/_transaction.haml [hunk 2]
```diff
@@ -1,10 +1,30 @@
 %tr
   %td= cleared_icon transaction.cleared
   %td= link_to transaction.date.to_s(:short_date), transaction
-  %td= transaction.account.name
+  %td= transaction.account.name if transaction.account_id
```
Comment (framallo) [inline app/views/transactions/_transaction.haml:4]: you could say if transaction.account
or transcation.account.try(:name)

Comment (framallo) [inline app/views/transactions/_transaction.haml:4]: every transaction should have an account, so you can validate it in the model

**Comment by sramallo** [comment]
Please see yesterday commits, Thanks

**Comment by framallo** [comment]
ok I added some comments

**Comment by sramallo** [comment]
Ok,
you can use @transaction.account.try(:name)

but besides this I have to add the validation in the model to does not come
any transaction without  account

2013/12/13 Federico Ramallo notifications@github.com

**Comment by sramallo** [comment]
Please check my commits, Thanks

**Comment by sramallo** [comment]
I added some commits, please see them, thanks

**Pull request closed without merging**